Assets/HoloToolkit/Input/Scripts/OnGazeLeaveEvent.cs

```diff
@@ -0,0 +1,21 @@
+﻿using UnityEngine;
+using UnityEngine.Events;
+
+public class OnGazeLeaveEvent : MonoBehaviour
+{
+    public UnityEvent Event;
+
+    void Start()
+    {
+        // dummy Start function so we can use this.enabled
```

Comment:
Those empty Start() functions are there so you have the toggle in the inspector meaning you can disable the component.
Maybe there's another way to have it without adding an empty Start or Update function I'll dig this.